mods/d2k/installer/d2k.yaml

```diff
@@ -0,0 +1,330 @@
+d2k: Dune 2000 (English)
+	Type: Disc
+	IDFiles:
+		MUSIC/AMBUSH.AUD: bd5926a56a83bc0e49f96037e1f909014ac0772a
```

Comment:
Using https://archive.org/details/dune-2000 I run into a weird case sensitivity problem on Linux:

![image](https://user-images.githubusercontent.com/756669/202766237-379ac653-dc95-45b2-a7b6-31a7bdc0984b.png)

where the ISO is `MUSIC/AMBUSH.AUD` but the mounted filesystem is `music/ambush.aud` so the check fails.

Reply:
Yup, thats what i addressed with the Case insensitive Part of part1?

Reply:
Oh i might have skipped the id Files! I'll check

Reply:
I've rechecked: the ID files are also checked case insensitive